### DISCUSSION TITLE
How to uninstall it?

### DISCUSSION THREAD

**Discussion by ceciver** [marlonrichert/zsh-autocomplete#333] [Q&A] [answered]
I just wanted to see how the autocompletion works, but I didn't like it and i'm unable to uninstall it

**Comment by marlonrichert** [comment]
How did you install it?

**Reply by truongsondq** [reply]
My personal comments:
There are many people who install the plugin according to the step-by-step instructions on a website, and website do not mention removal.
Good plug-ins have uninstall instructions, for example Oh-my-zsh:



> ## Uninstalling Oh My Zsh
> 
> Oh My Zsh isn't for everyone. We'll miss you, but we want to make this an easy breakup.
> 
> If you want to uninstall `oh-my-zsh`, just run `uninstall_oh_my_zsh` from the command-line. It will remove itself and revert your previous `bash` or `zsh` configuration.

**Comment by marlonrichert** [comment, marked as answer]
I've now added [uninstall instructions to the Readme](https://github.com/marlonrichert/zsh-autocomplete#installing--updating).
